l10n-pocket/en/nav.ftl

```diff
@@ -6,6 +6,7 @@
 
 # pocket/includes/nav.html
 # pocket/includes/mobile-nav.html
+# pocket/includes/platform-nav.html
```

Comment:
Again these aren't tied to any specific strings. Should they be group level comments throughout the file?